drivers/drivers.go

```diff
@@ -96,6 +96,8 @@ type OSInfo_StorageType int32
 type S3OSInfo struct {
 	// Host to use to connect to S3
 	Host string `protobuf:"bytes,1,opt,name=host,proto3" json:"host,omitempty"`
+	// Bucket where the object is stored
+	Bucket string `json:"bucket,omitempty"`
```

Comment:
I'm not sure whether to add the protobuf declaration, the json isn't needed for my intended use case either.. maybe they were added just in case it was ever needed